src/doc/rustc/src/codegen-options/index.md

```diff
@@ -209,6 +209,19 @@ Note that while the `-C instrument-coverage` option is stable, the profile data
 format produced by the resulting instrumentation may change, and may not work
 with coverage tools other than those built and shipped with the compiler.
 
+## jump-tables
+
+This option is used to allow or prevent the codegen backend from creating
+jump tables when lowering switches.
```

Comment:
Discussion: Hm, what happens if a different cg backend is selected?

cc @GuillaumeGomez @antoyo

Reply:
Should this be implemented for the other backends before stabilizing?

Reply:
Cranelift doesn't have support emission of jump tables for the `br_table` instruction. Avoiding emitting `br_table` itself may be feasible, but would require a cranelift-frontend change. FWIW cg_clif silently ignores many of these flags already.

Reply:
> (assuming Antoni is okay with doing it here rather than the submodule repo to avoid dancing also dancing around the name change).

I'm OK with this.

Reply:
Adding support to the gcc backend seems straightforward. What is the difference between using `add_driver_option` vs `add_command_line_option`? The former did inhibit jump tables when I tested it.

Reply:
Some options won't work if not using the correct function. If it works, all is good.

Reply:
> Should this be implemented for the other backends before stabilizing?

Given that only the LLVM codegen backend is "stably" supported, it is not a stabilization blocker.